### PR TITLE
Update breadcrumbs to use HashRouter

### PR DESCRIPTION
**Pull request by benq66**
- to simplify breadcrumbs management a "use-react-router-breadcrumbs" package was used
- some additional minor refactoring was conducted

### CODE REVIEW

File: h5p-bildetema/package.json [hunk 1]
```diff
@@ -46,6 +47,7 @@
     "@types/react": "^18.0.12",
     "@types/react-dom": "^18.0.5",
     "@vitejs/plugin-react": "^1.3.2",
+    "esbuild": "^0.14.43",
```
Comment (boyum) [inline h5p-bildetema/package.json:50]: What is this needed for?

**Comment by benq66** [comment]
Oh, forgot to remove memory router... Whoops. And need to update the stories 😅

File: h5p-bildetema/package.json (hunk 1 above)
Comment (benq66) [reply h5p-bildetema/package.json:50]: I think I installed it when I got the following message when trying to run Storybook:
```
The package "esbuild-windows-64" could not be found, and is needed by esbuild.

If you are installing esbuild with npm, make sure that you don't specify the
"--no-optional" flag. The "optionalDependencies" package.json feature is used
by esbuild to install the correct binary executable for your current platform.
The package "esbuild-windows-64" could not be found, and is needed by esbuild.
```
Then I found this topic and it was suggested to install esbuild - https://github.com/evanw/esbuild/issues/1646#issuecomment-1069353130

File: h5p-bildetema/src/components/Breadcrumbs/Breadcrumbs.tsx [hunk 2]
```diff
@@ -1,35 +1,56 @@
...
 import { Link } from "react-router-dom";
+import useBreadcrumbs from "use-react-router-breadcrumbs";
+import { BreadcrumbsArrowIcon } from "../Icons/Icons";
 import styles from "./Breadcrumbs.module.scss";
 
+const routes = [{ path: "/nb", breadcrumb: "Norsk (bokmål)" }];
+
 export type BreadcrumbsProps = {
-  breadcrumbs: {
+  breadcrumbsTest?: {
```
Comment (boyum) [inline h5p-bildetema/src/components/Breadcrumbs/Breadcrumbs.tsx:10]: Rename this to `breadCrumbs`?

Comment (benq66) [reply h5p-bildetema/src/components/Breadcrumbs/Breadcrumbs.tsx:10]: \+

**Review by boyum — APPROVED** [review]

**Pull request merged**